parser_library/test/workspace/workspace_test.cpp

```diff
@@ -75,17 +79,21 @@ TEST_F(workspace_test, parse_lib_provider)
 
     file_mngr.add_processor_file("test\\library\\test_wks\\correct");
 
-    context::hlasm_ctx_ptr ctx_1, ctx_2;
     lsp::lsp_ctx_ptr lsp_ptr = std::make_shared<lsp::lsp_context>();
-#if _WIN32
-    ws.did_open_file("test\\library\\test_wks\\correct");
-    ctx_1 = std::make_shared<context::hlasm_context>("test\\library\\test_wks\\correct");
-    ctx_2 = std::make_shared<context::hlasm_context>("test\\library\\test_wks\\correct");
-#else
-    ws.did_open_file("test/library/test_wks/correct");
-    ctx_1 = std::make_shared<context::hlasm_context>("test/library/test_wks/correct");
-    ctx_2 = std::make_shared<context::hlasm_context>("test/library/test_wks/correct");
-#endif
+    auto [ctx_1, ctx_2] = [&ws]() {
+        if (platform::is_windows())
+        {
+            ws.did_open_file("test\\library\\test_wks\\correct");
+            return std::make_pair(std::make_shared<context::hlasm_context>("test\\library\\test_wks\\correct"),
```

Comment:
maybe we could use `std::filesystem::path("test/library/test_wks/correct").native()` in such cases (if path::native is not broken in emscripten)